### DISCUSSION TITLE
Direction on RPGLE Free Extension

### DISCUSSION THREAD

**Discussion by SJLennon** [codefori/vscode-ibmi#264] [General]
I have installed the RPGLE Free extension for experimentation.  Are there plans to include it automatically?

(I've been pleased with what it does for fixed D-specs and most of my other statements are already free form.  

However, I did find some old code that was totally fixed and found that the extension seems to die without explanation when it finds a plist or a parm statement.  Granted these are hard to convert and possibly can't or shouldn't be done automatically. Perhaps best just to leave them as is.)

**Comment by worksofliam** [comment]
I assume you mean this: https://marketplace.visualstudio.com/items?itemName=BrianJGarland.vscode-rpgfree

There is no plan to include it in this, but it is part of the Extension Pack I am working on:

* https://github.com/halcyon-tech/ibm-i-development-pack
* https://marketplace.visualstudio.com/items?itemName=HalcyonTechLtd.ibm-i-development-pack

The extension pack needs some work: e.g. a logo and a readme
